### PR TITLE
Add new widgets to Marketplace

### PR DESCRIPTION
**Pull request by dmahmouali**
### Changes
- change Marketplace package home page to be public and the authentication on chatflows and listing user's apps.
- Added new apps.
- Update the wiki links and images.
- 
image

### Related Issues

[TF NOW: Add new widgets to Marketplace](https://github.com/threefoldtech/js-sdk/issues/1466)

### CODE REVIEW

File: jumpscale/packages/marketplace/frontend/components/Home.vue [hunk 1]
```diff
@@ -84,9 +86,17 @@ module.exports = {
           return obj;
         }, {});
     },
+    getCurrentUser() {
```
Comment (abom) [inline jumpscale/packages/marketplace/frontend/components/Home.vue:89]: why don't we redefine the user here? cannot we pass it from parent component or get it from a shared state instead?

File: jumpscale/packages/marketplace/package.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name = "marketplace"
-is_auth = true
+is_auth = false
```
Comment (m-motawea) [inline jumpscale/packages/marketplace/package.toml:2]: I think you can remove now the configuration of the package actors here
https://github.com/threefoldtech/js-sdk/blob/a03dcf35dfdf1b75611532056b47fb0658d8c450/jumpscale/packages/marketplace/package.py#L18

**Comment by sasha-astiadi** [comment]
where is the 'my workload' button next to deploy button?

if clicked should display all deployed solution. if clicked and not logged in, ask user to login.

**Review by waleedhammam — APPROVED** [review]
Thanks <3

**Comment by xmonader** [comment]
> where is the 'my workload' button next to deploy button?
> 
> if clicked should display all deployed solution. if clicked and not logged in, ask user to login.

Why would you should "my workloads" to non-logged in user? 

if user logs in 
image

**Comment by xmonader** [comment]
@dmahmouali Please make sure "all of the coming soon solutions" come after all of the "deployable" ones

**Review by ranatrk — APPROVED** [review]

File: jumpscale/packages/marketplace/frontend/components/Home.vue [hunk 3]
```diff
@@ -52,7 +52,7 @@
                 :apps="section.apps"
                 :titletooltip="section.titleToolTip"
                 :solutioncount="solutionCount"
-                :user="user"
+                :logedin="logedin"
```
Comment (xmonader) [inline jumpscale/packages/marketplace/frontend/components/Home.vue:55]: loggedin*

File: jumpscale/packages/marketplace/frontend/components/Home.vue [hunk 4]
```diff
@@ -74,7 +74,7 @@ module.exports = {
       solutionCount: {},
       selectedObject: {},
       sections: SECTIONS,
-      user: null,
+      logedin: this.$route.params.logedin,
```
Comment (xmonader) [inline jumpscale/packages/marketplace/frontend/components/Home.vue:77]: loggedin*

File: jumpscale/packages/marketplace/frontend/components/base/SolutionsSection.vue [hunk 5]
```diff
@@ -49,7 +49,7 @@
           <v-btn v-if="app.disable" text medium disabled>Coming Soon</v-btn>
           <div v-else>
             <v-btn text medium color="green" @click.stop="openChatflow(app.type)">Deploy</v-btn>
-            <v-btn v-if="user" text medium @click.stop="viewWorkloads(app)">My Workloads</v-btn>
+            <v-btn v-if="logedin" text medium @click.stop="viewWorkloads(app)">My Workloads</v-btn>
```
Comment (xmonader) [inline jumpscale/packages/marketplace/frontend/components/base/SolutionsSection.vue:52]: loggedin*

File: jumpscale/packages/marketplace/frontend/components/base/SolutionsSection.vue [hunk 6]
```diff
@@ -65,7 +65,7 @@ module.exports = {
     titletooltip: String,
     apps: Object,
     solutioncount: Object,
-    user: Boolean,
+    logedin: Boolean,
```
Comment (xmonader) [inline jumpscale/packages/marketplace/frontend/components/base/SolutionsSection.vue:68]: loggedin*

**Review by rkhamis — APPROVED** [review]

**Pull request merged**